connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -2461,14 +2450,65 @@ ble_error_t Gap::startAdvertising(
         }
 
         _interruptible_sets.set(LEGACY_ADVERTISING_HANDLE);
-    }
 
-    _pending_sets.set(handle);
+        _pending_sets.set(handle);
+    }
 
     return error;
 }
 #endif
 
+#if BLE_FEATURE_EXTENDED_ADVERTISING
+void Gap::process_enable_queue()
+{
+    tr_info("Evaluating pending advertising sets to be started");
+    if (!_advertising_enable_command_params.number_of_handles) {
+        /* no set pending to be enabled */
+        tr_debug("No sets to be enabled");
+        return;
+    }
+
+    for (size_t i = 0; i < BLE_GAP_MAX_ADVERTISING_SETS; ++i) {
+        if (_pending_sets.get(i)) {
+            /* we have to wait until nothing is pending */
+            tr_debug("Cannot enable sets as pending sets present");
```

Comment:
`tr_info` ? That could cause the failure if that function is called directly.

Reply:
it will be retried and is part of normal operation but I don't mind info